java/org/apache/catalina/realm/JNDIRealm.java

```diff
@@ -966,7 +967,7 @@ private String[] getCipherSuitesArray() {
             containerLog.warn(sm.getString("jndiRealm.emptyCipherSuites"));
             this.cipherSuitesArray = null;
         } else {
-            this.cipherSuitesArray = cipherSuites.trim().split("\\s*,\\s*");
+            this.cipherSuitesArray = StringUtils.splitCommaSeparated(cipherSuites.trim());
             if (containerLog.isTraceEnabled()) {
```

Comment:
Is this trim necessary here? I know it was there before but it looks redundant.

Reply:
It turns out its doubly-redundant. It's "gone" but it's still there because the condition above also needs to trim the string to check for empty-string. But now we are only trimming once in this method instead of twice.